src/status_im/contexts/communities/actions/share_community/style.cljs

```diff
@@ -0,0 +1,61 @@
+(ns status-im.contexts.communities.actions.share-community.style
+  (:require [quo.foundations.colors :as colors]))
+
+(def flex-direction-row {:flex-direction :row})
+(def horizontal-padding 20)
+(def vertical-padding 12)
+(def gradient-cover-padding 20)
+(def qr-code-padding 12)
+
+(def header-container
+  {:padding-horizontal horizontal-padding
+   :padding-vertical   vertical-padding})
+
+(def scan-notice
+  {:color        colors/white-70-blur
+   :margin-top   16
+   :margin-left  :auto
+   :margin-right :auto})
+
+(defn community-name
+  [thumbnail]
+  {:margin-left (when thumbnail 8)})
+
+(def qr-code-wrapper
+  {:padding-horizontal horizontal-padding
+   :margin-top         8})
+
+(defn qr-code-size
+  [total-width]
+  (- total-width (* gradient-cover-padding 2) (* qr-code-padding 2)))
+
+(defn gradient-cover-size
+  [total-width]
+  (- total-width (* gradient-cover-padding 2)))
+
+(defn gradient-cover-wrapper
+  [width]
+  {:width         (gradient-cover-size width)
+   :border-radius 16
+   :margin-left   20
+   :height        "100%"})
+
+(def share-button-container {:justify-self :flex-end})
+
+(def qr-top-wrapper
+  {:margin          12
+   :margin-bottom   0
+   :flex-direction  (:flex-direction flex-direction-row)
```

Comment:
This is a bit odd. Could be just `:flex-direction :row` because you're already using a var with the word *row* in it, so the abstraction to get the flex direction from the var is unnecessary.